server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1024,12 +1024,13 @@ private SortedMap<TServerInstance,TabletServerStatus> gatherTableInformation(
         }
       }));
     }
-    final long timeToWaitForCompletion = Math.max(10000, rpcTimeout / 3);
+    // wait at least 10 seconds
+    final long timeToWaitNanos =
+        MILLISECONDS.toNanos(Math.max(SECONDS.toNanos(10), MILLISECONDS.toNanos(rpcTimeout) / 3));
```

Comment:
These are already nanos.

```suggestion
    final long timeToWaitNanos = Math.max(SECONDS.toNanos(10), MILLISECONDS.toNanos(rpcTimeout) / 3);
```

Reply:
fixed in fa538b0b3e

Reply:
As you predicted, my formatting wasn't quite right. One thing that might work, to make the line shorter, is to rename the variable from `timeToWaitNanos` to `nanosToWait`. That might be enough to make it fit on one line.